test/uk/gov/hmrc/tradergoodsprofilesrouter/service/UuidServiceSpec.scala

```diff
@@ -14,17 +14,22 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.tradergoodsprofilesrouter.controllers
+package uk.gov.hmrc.tradergoodsprofilesrouter.service
 
-import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
-import play.api.mvc.{Action, AnyContent, ControllerComponents}
-import javax.inject.{Inject, Singleton}
-import scala.concurrent.Future
+import org.scalatestplus.play.PlaySpec
 
-@Singleton()
-class MicroserviceHelloWorldController @Inject() (cc: ControllerComponents) extends BackendController(cc) {
+class UuidServiceSpec extends PlaySpec {
 
-  def hello(): Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok("Hello world"))
+  private val uuidService = new UuidService
+
+  "UuidService" should {
```

Comment:
I would not bother to test this as we are testing a Java function here and it does not need to be tested.